### DISCUSSION TITLE
Frage zum Status Wording

### DISCUSSION THREAD

**Discussion by KingLuy** [camueller/SmartApplianceEnabler#497] [Allgemein]
Hallo Herr Müller,

eine Frage bezüglich des Wordings der Status Ampel. Ich bin etwas verwirrt durch das Wort blinkt. Soll die Ampel wirklich blinken, also an, aus, an, aus, .... oder ist hier leuchten gemeint. Weil bei mir blinkt nichts.
Ist Zustand:
Das Gerät kann jederzeit (wieder) eingeschaltet werden, weil eine Bedarfsanforderung vorliegt (bei blinkender Leuchte lediglich für Überschussenergie) und der frühestmögliche Einschaltzeitpunkt erreicht ist.
Das Gerät ist eingeschaltet. Falls die grüne Leuchte blinkt, wird Überschussenergie verwendet.

Vorschlag:
Das Gerät kann jederzeit (wieder) eingeschaltet werden, weil eine Bedarfsanforderung vorliegt (bei eingeschalteter Leuchte lediglich für Überschussenergie) und der frühestmögliche Einschaltzeitpunkt erreicht ist.
Das Gerät ist eingeschaltet. Falls die grüne Leuchte eingeschaltet ist, wird Überschussenergie verwendet.

p.s. ist wirklich keine Kritik, ist ja nur ein Vorschlag wenn es wirklich nicht blinken sollte.

Grüße
King Luy

**Comment by camueller** [comment]
>  Soll die Ampel wirklich blinken, also an, aus, an, aus, ....

Ja, sonst hätte ich das nicht so geschrieben :-)

**Reply by KingLuy** [reply]
okay, bei mir blinkt aber nichts. Fehlt da ggf. irgend etwas. PV Überschuss habe ich gerade 4kW

**Comment by camueller** [comment]
Wenn nichts blinkt, hast Du wohl keinen Einschaltbefehl vom SHM bekommen.

**Reply by KingLuy** [reply]
Also Ampel ist grün und die Shelly Plug S ist an gegangen.

**Comment by KingLuy** [comment]
Hallo Herr Müller,
nochmal eine Frage in dem Thema. Woran kann es liegen das die Ampel nicht blinkt?
Es hatte damals dann funktioniert. Dann ist mir mein Raspberry gestorben und ich hab es neu aufgesetzt. Jetzt blinkt die Ampel wieder nicht.

Und wenn ich keinen Überschuss habe also aus dem Netz ziehe schaltet er den Shelly nicht mehr aus.

Grüße

**Reply by KingLuy** [reply]
Ich habe das Problem gefunden. Bei Schaltzeiten muss 00:00 als Mindestlaufzeit angegeben werden damit es blinkt.
